plugins/zenoh-plugin-rest/src/lib.rs

```diff
@@ -34,6 +34,7 @@ use zenoh::plugins::{RunningPluginTrait, ZenohPlugin};
 use zenoh::prelude::r#async::*;
 use zenoh::query::{QueryConsolidation, Reply};
 use zenoh::runtime::Runtime;
+use zenoh::sample_builder::ValueBuilderTrait;
```

Comment:
I suggest to add the builder traits directly in the prelude.

Reply:
done